src/reading.rs

```diff
@@ -1114,8 +1114,8 @@ pub mod async_api {
 	}
 
 	/**
-	Async packet reading functionality.
```

Comment:
Why are you replacing tabs with spaces here?

Reply:
Clippy tell it is better but i can revert it if you prefer.